src/Manager/ConfigManager.php

```diff
@@ -45,12 +45,11 @@ class ConfigManager {
 	const LEVEL_VERSION = 0;
 	const TRANSLATION_VERSION = 2;
 	const LANG_FILE_VERSION = [
-		"en_EN" => 7,
-		"fr_FR" => 5,
-		"es_SPA" => 2,
-		"tr_TR" => 1,
-		"pt_BR" => 1,
-		"tr_TR" => 1
+		"en_EN" => 8,
```

Comment:
Why change the translation versions when you don't modify the files anymore?

Reply:
I forgot to put the old file back